src/AppBundle/Resources/views/layout.html.twig

```diff
@@ -153,7 +153,8 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/marked/0.3.5/marked.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.textcomplete/0.2.2/jquery.textcomplete.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.10.6/moment.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/highcharts/4.1.7/highcharts.js"></script>
+    <script src="https://code.highcharts.com/highcharts.js"></script>
```

Comment:
This doesn't link to a specific version. Would prefer if it did so it doesn't randomly break if they update things.